modules/transport.js

```diff
@@ -374,7 +374,7 @@ Transport.prototype.onNewBlock = function (block, broadcast) {
 	if (broadcast) {
 		modules.system.update(function () {
 			if (!__private.broadcaster.maxRelays(block) && !modules.loader.syncing()) {
-				modules.peers.list({}, function (err, peers) {
+				modules.peers.list({normalized: false}, function (err, peers) {
```

Comment:
it is currently in place. Normalized set to `true` by default to remain previous functionalities:
https://github.com/LiskHQ/lisk/pull/683/files#diff-5ad295c030c66fe97f92104bd5d68cefR114